src/__tests__/customerDb.test.js

```diff
@@ -139,4 +148,59 @@ describe('Customers model tests', () => {
 
     expect(customerObj.companyIds).toEqual(expect.arrayContaining(companyIds));
   });
+
+  test('removeCustomer', async () => {
+    const customer = await customerFactory({});
+
+    await internalNoteFactory({
+      contentType: COC_CONTENT_TYPES.CUSTOMER,
+      contentTypeId: customer._id,
+    });
+    await conversationFactory({
+      customerId: customer._id,
+    });
+    await conversationMessageFactory({
+      customerId: customer._id,
+    });
+
+    await Customers.removeCustomer(customer._id);
+
+    expect(
+      await InternalNotes.find({
+        contentType: COC_CONTENT_TYPES.CUSTOMER,
+        contentTypeId: customer._id,
+      }),
+    ).toHaveLength(0);
+    expect(await Conversations.find({ customerId: customer._id })).toHaveLength(0);
+    expect(await ConversationMessages.find({ customerId: customer._id })).toHaveLength(0);
+  });
+
+  test('Merge customers', async () => {
```

Comment:
Why are you not testing actual logics ?. Where is checks of merged customer fields ?